grid/project-management/tasks/profile.supr

```diff
@@ -0,0 +1,71 @@
+"""
+Manage tasks in project management systems.
+"""
+name = "project-management/tasks"
+version = "1.0.0"
+
+"""
+Create a new task.
+"""
+usecase CreateTask {
+	input {
+
+    """
+    projects
+    An array of IDs of projects this task belongs to.
+    """
+    projects! [string]!
+
+		"""
+		label
+		Textual label of the task.
+		"""
+		label! string!
```

Comment:
Is label Asana-specific lingo? I think I'd rather prefer `title` which seems more common?

Reply:
Nope, just my personal idiosyncrasy, I suppose. ("`label`/`description`" is a combination I feel like I often encounter).
FWIW, Asana calls it a `name` which didn't sit right with me.
I think `title` works fine though.